### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by anzilparviz29**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by anzilparviz29 — COMMENTED** [review]
Nice work

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (anzilparviz29) [inline index.html:50]: good

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (habibrahman29) [inline index.html:50]: iu

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (anzilparviz29) [inline index.html:48]: Nice work

File: index.html (hunk 2 above)
Comment (habibrahman29) [inline index.html:50]: gooddd

**Review by anzilparviz29 — COMMENTED** [review]
iu

File: index.html [hunk 4]
```diff
@@ -27,7 +27,7 @@
   }
 </style>
 
-<h1>Tic Tac Toe</h1>
+<h1>Tic Tac Toe 2</h1>
```
Comment (anzilparviz29) [inline index.html:30]: Nice

File: index.html (hunk 1 above)
Comment (anzilparviz29) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

**Review by anzilparviz29 — COMMENTED** [review]
yes

**Pull request merged**